skeleton/pipeline.mk

```diff
@@ -1,14 +1,15 @@
-################################################################################
+#
 #
```

Comment:
While we are reviewing, does a separate `skeleton/pipeline.mk` make sense? I am inexperienced with Makefiles.

Reply:
Yes, it makes perfect sense. But it would be out of scope for this PR so we will fix/enhance in separate PRs.